script.js

```diff
@@ -0,0 +1,94 @@
+var x = true;
+var o =false;
+var valid = false;
+var xchosen = [];
+var ochosen = [];
+
+//reset onClick
+$("#reset").click(reset);
+
+function reset(){
+  $("td").removeClass("X").removeClass("O");
+  $("td").html("");
+  $(".playerTurn").html("It is X's turn")
+  x = true;
+  o = false;
+}
+
+//playerturn
+function changeTurn(){
```

Comment:
What is this doing? As far as I can see it sets both x and o to be false, which means it can't be either player's turn.

Reply:
it sets x to be true and o to be false